agorauikit_android/src/main/java/io/agora/agorauikit_android/AgoraVideoViewer.kt

```diff
@@ -228,6 +267,345 @@ open class AgoraVideoViewer: FrameLayout {
         agkit.enableVideo()
         agkit.setVideoEncoderConfiguration(VideoEncoderConfiguration())
     }
+
+    private fun initAgoraRtm(context: Context) {
+        try {
+            this.agRtmClient = RtmClient.createInstance(context, connectionData.appId,
+                object : RtmClientListener {
+                    override fun onConnectionStateChanged(state: Int, reason: Int) {
+                        println("RTM Connection State Changed. state: $state, reason: $reason")
+                    }
+
+                    override fun onMessageReceived(rtmMessage: RtmMessage, peerId: String?) {
+                        println("Peer message received from ${peerId.toString()}, " + rtmMessage.text.toString())
+                        messageReceived(rtmMessage.text)
+                    }
```

Comment:
How would a dev access these delegate methods and catch incoming messages of their own?